dbos_transact/fastapi.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any, Callable
+
+from fastapi import FastAPI
+from fastapi import Request as FastAPIRequest
+
+from .context import DBOSContextEnsure, SetWorkflowUUID, assert_current_dbos_context
+from .logger import dbos_logger
+
+
+class Request:
+    """
+    A serializable subset of the FastAPI Request object
+    """
+
+    def __init__(self, req: FastAPIRequest):
```

Comment:
Can we make this the same as transact-ts? https://docs.dbos.dev/api-reference/contexts#ctxtrequest

Reply:
That one was Koa-based, this is FastAPI-based